config/rbac/role.yaml

```diff
@@ -129,6 +130,7 @@ rules:
   resources:
   - signers
   verbs:
+  - Approve
   - approve
```

Comment:
Where is this coming from? This isnt just `release prep` when this is a functionality change.

Reply:
I don't think that's required.

Reply:
Seems to have been generated when I ran `make bundle`

Reply:
@sebsoto @mansikulkarni96 I've manually removed the role changes.